gh-prs

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+# https://github.com/dlvhdr/gh-prs/pull/11#discussion_r756341767
+LATEST_VERSION=$(curl --silent "https://api.github.com/repos/dlvhdr/gh-prs/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
I think you can just omit the tag from the `gh release download` command.
Having a curl request at the top of the script will slow it down.

Reply:
you got it